scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -563,6 +563,13 @@ public String getTaskServerLogsByTag(String id, String taskTag)
         return client.getJobs(offset, limit, filterCriteria, sortParameters);
     }
 
+    @Override
+    public List<JobInfo> getJobsInfoList(List<String> jobsId)
+            throws PermissionException, NotConnectedException, UnknownJobException {
+        renewSession();
+        return client.getJobsInfoList(jobsId);
+    }
+
     @Override
     public List<SchedulerUserInfo> getUsers() throws NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)